### PR TITLE
opening model completed

### PR DESCRIPTION
**Pull request by sanyog96**
### Changes in this PR:
opening model

### Reference GitHub issues:
#15

### CODE REVIEW

File: app/models/Education.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    default: CONSTANTS.ENUMS.USER.PERFORMANCE_SCALE.CGPA,
+                },
+                value: {
+                    type: Number,
+                },
+            },
+            school: {
+                type: String,
+            },
+            required: true,
```
Comment (sanyog96) [inline app/models/Education.js]: does this "required" tag conditions on all of inner attributes that they are all needed to be entered or need to put required in each of it ?????

File: app/models/JobOpening.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        },
+        start_date: {
+            type: Date,
+            required: true,
+        },
+        end_date: {
+            type: Date,
+            required: true,
+        },
+        interviews: [
```
Comment (Jo-ravar) [inline app/models/JobOpening.js]: Change it to `interviewees`

File: app/models/JobDescription.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        organization: {
+            type: String,
+            required: true,
+        },
+        location: {
+            type: String,
+            required: true,
+        },
+        start_date: {
+            type: String,
```
Comment (Jo-ravar) [inline app/models/JobDescription.js]: This should be of `date` type

File: app/models/JobDescription.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            required: true,
+        },
+        currently_working: {
+            type: Boolean,
+        },
+        end_date: {
+            type: String,
```
Comment (Jo-ravar) [inline app/models/JobDescription.js]: This should be of `date` type.

File: app/models/Interviewee.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                },
+                score: {
+                    type: String,
+                },
+                date: {
+                    type: String,
+                },
+            },
+        ],
+        projects: [
```
Comment (Jo-ravar) [inline app/models/Interviewee.js]: Make different schema for `projects`  it will make code clean.

File: app/models/Interviewee.js [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+                    type: String,
+                },
+                description: {
+                    type: String,
+                    maxlength: 150,
+                },
+            },
+        ],
+        certifications: [
+            {
```
Comment (Jo-ravar) [inline app/models/Interviewee.js]: Make different file for `certification` schema will make code clean.

File: app/models/JobDescription.js (hunk 3 above)
Comment (sanyog96) [reply app/models/JobDescription.js]: interviewee only needs to enter the month from when he/she has started working for a job or switched from a job.
if `Date` type is set then user needs to enter specific date to be saved in database which might look difficult for many of them to enter.

File: app/models/JobDescription.js (hunk 4 above)
Comment (sanyog96) [reply app/models/JobDescription.js]: interviewee only needs to enter the month from when he/she has started working for a job or switched from a job.
if `Date` type is set then user needs to enter specific date to be saved in database which might look difficult for many of them to enter.

**Pull request merged**